Specification/src/Ardalis.Specification/Exceptions/DuplicateSkipException.cs

```diff
@@ -6,7 +6,7 @@ namespace Ardalis.Specification
 {
     public class DuplicateSkipException : Exception
     {
-        private const string message = "Duplicate use of the Skip(). Ensure you don't use both Paginate() and Skip() in the same specification!";
+        private const string message = "Duplicate use of the Skip(). Ensure you don't use Skip() in the same specification!";
```

Comment:
Perhaps the message should be "Duplicate use of Skip(). Ensure you don't use Skip() more than once in the same specification!".